08_files_lab/01.py

```diff
@@ -1,6 +1,12 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+import sys
+import os 
+
+(_, file1, file2) = sys.argv
+
+with open ("./" + file1, 'r') as fin:
+    with open ("./" + file2, 'w') as fout:  
+        for line in fin:
+            fout.write(line)
+            print line
+
+#should I delete file1 contents or just copy the lines to file2?
```

Comment:
Maybe I'm misunderstanding but when I'm writing full path (without the file name because the user need to select it), I'm limiting the user to a specific path, 
lets say:

**with open( "c:\\" + file1 .......)**
so the limitation here is that the program is searching files only in disk C